features/canvas-2d-context-colorspace.yml

```diff
@@ -0,0 +1,6 @@
+name: Canvas 2D `colorSpace` parameter
```

Comment:
See comment above about the `alpha` parameter.

Reply:
I went back and forth on the naming for these a handful of times. I went with "Canvas 2D \`colorSpace\` parameter" to try to be consistently for all of the special context parameters (colorSpace, alpha, desynchronized, willReadFrequently). I think the naming makes it clear its for the canvas 2d rendering context, and what parameter it is.